docker-compose.yml

```diff
@@ -15,5 +14,11 @@ services:
     environment:
       MYSQL_DATABASE: starchart
       MYSQL_USER: starchart
-      MYSQL_PASSWORD: starchart_password 
+      MYSQL_PASSWORD: starchart_password
       MYSQL_ROOT_PASSWORD: root_password
+
+  dns:
+    container_name: route53_mock
+    image: motoserver/moto:4.1.1
+    ports:
+      - "5050:5000"
```

Comment:
Let's use `5053` to give a nod to this being a Route53 instance.

Reply:
I like the suggestion!